azure-pipelines/nightly-snapshot-releases/Latest Common VSTS Release.yml

```diff
@@ -1,108 +1,34 @@
-# 'Allow scripts to access the OAuth token' was selected in pipeline.  Add the following YAML to any steps requiring access:
-#       env:
-#           MY_ACCESS_TOKEN: $(System.AccessToken)
-# Variable 'android:serverUrl' was defined in the Variables tab
-# Variable 'ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME' was defined in the Variables tab
-# Variable 'test_repo_branch' was defined in the Variables tab
-# Variable 'test_repo_dir' was defined in the Variables tab
-# Variable Group 'Android Common Variables' was defined in the Variables tab
-# Cron Schedules have been converted using UTC Time Zone and may need to be updated for your location
+# File: azure-pipelines\nightly-snapshot-releases\Latest Common VSTS Release.yml
+# Description: Assemble & publish Latest common release nightly to internal feed
+# https://identitydivision.visualstudio.com/Engineering/_packaging?_a=feed&feed=AndroidADAL
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger: none
+pr: none
+
 resources:
-  pipelines:
-  - pipeline: Nightly ADAL VSTS Release
-    source: Nightly ADAL VSTS Release
-    trigger:
-      branches:
-        include:
-        - dev
+# TODO uncomment once we have a valid artifact in the pipeline
+#  pipelines:
+#  - pipeline: Android\Nightly Snapshot Releases\Nightly ADAL VSTS Release
+#    project: Engineering
+#    source: Nightly ADAL VSTS Release
```

Comment:
I see this file was referring to the `LATEST` pipeline for `common`...but I see here it says nightly adal isntead...which one is correct?

Reply:
is refereeing to this [pipeliene])(https://identitydivision.visualstudio.com/IDDP/_build?definitionId=632) to the yaml version once is up, I  will uncomment this 
https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1384923

Reply:
Actually this doesn't make sense to me. Why is ADAL triggering builds of common? It should be the other way around...common should be triggering builds of MSAL / ADAL etc...